EIPS/eip-1328.md

```diff
@@ -25,17 +25,30 @@ WalletConnect request URI with the following parameters:
     version       = 1*DIGIT
     parameters    = parameter *( "&" parameter )
     parameter     = key "=" value
-    key           = "bridge" / "key"
+    key           = STRING
     value         = STRING
 
 ### Semantics
 
-Required parameters are dependent on the Walletconnect protocol version which currently includes the `key`, hex string of symmetric key, and `bridge`, encoded url of the bridge used for establishing the connection.
+Required parameters are dependent on the Walletconnect protocol version:
+
+For v1.0 protocol the required parameters are:
+- `key` - symmetric key used for encryption
+- `bridge` - url of the bridge server for relaying messages
+
+For v2.0 protocol the required parameters are:
```

Comment:
I think we should do For version=2 as we only have a major version in this versioning scheme here